File: Modul-6/Py/PRAK601-2310817110006-Adrian_Bintang_Saputera.py
baris, kolom = map(int, input("Input\n").split())
nilai_input = list(map(int, input().split()))

matriks_midoriya = [nilai_input[i:i+kolom] for i in range(0, len(nilai_input), kolom)]

print("\nOutput")
for baris in matriks_midoriya:
    print(*baris)

File: Modul-6/C/PRAK601-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
  int baris, kolom;

  printf("Input\n");
  scanf("%d %d", &baris, &kolom);

  int matrix[baris][kolom];

  for (int i = 0; i < baris; i++)
    for (int j = 0; j < kolom; j++)
      scanf("%d", &matrix[i][j]);

  printf("\nOutput\n");
  for (int i = 0; i < baris; i++, printf("\n"))
    for (int j = 0; j < kolom; j++)
      printf("%d ", matrix[i][j]);

  return 0;
}